service/templates/waterfall.html

```diff
@@ -27,6 +27,10 @@
 {{end}}
 
 {{define "content"}}
+<div class="banner warning">
+  The waterfall page will be deprecated on May 20th. Please use the <a href="{{.NewUILink}}">new Waterfall page</a> instead.
+</div>
```

Comment:
The styling did not apply make sure it looks like a banner and you are using the correct classes